src/test/features/staging-spec.ts

```diff
@@ -114,6 +114,20 @@ suite('ModelViewerElementBase with StagingMixin', () => {
                                // rAF though
         expect(element.turntableRotation).to.be.greaterThan(turntableRotation);
       });
+
+      test('pauses rotate after user interaction', async () => {
+        const {turntableRotation: initialTurntableRotation} = element;
+
+        element.dispatchEvent(new CustomEvent('user-camera-orbit-change'));
+
+        await timePasses(50);  // An arbitrary amount of time, greater than one
+                               // rAF though
+        expect(element.turntableRotation).to.be.equal(initialTurntableRotation);
+
+        await timePasses(AUTO_ROTATE_DELAY_AFTER_USER_INTERACTION);
```

Comment:
I do not like this because it does like increase the duration it takes to run tests. But I'm not sure how to handle this easily.

Reply:
Yeah I was thinking about this also. That in the 2nd PR we can shorten it. I do like how Jest has

https://jestjs.io/docs/en/timer-mocks#run-all-timers

Makes it super handy for this kind of thing.

Reply:
Yeah, our testing situation very much feels like a throwback. I hope we can find the time to upgrade it a bit in the near future 🙏